src/resources/views/inc/menu.blade.php

```diff
@@ -19,7 +19,7 @@
 
       <!-- <li><a href="{{ url('/') }}"><i class="fa fa-home"></i> <span>Home</span></a></li> -->
 
-        @if (Auth::guest())
+        @if ($backpackAuth->guest())
```

Comment:
From the [AuthComposer](https://github.com/jorenvanhee/Base/blob/different-user-model-issue-fix/src/app/Http/ViewComposers/AuthComposer.php).